src/app/helptext/system/support.ts

```diff
@@ -66,6 +66,13 @@ export const helptext_system_support = {
     validation: [Validators.required, Validators.email]
   },
 
+  cc: {
+    placeholder: T("CC"),
+    tooltip: T("Enter one or more email addresses which should receive copies of support correspondence \
+ on this issue. Separate addresses with a comma."),
+    err: T('All emails must be in valid form.')
```

Comment:
Adding a suggestion that includes correct email formatting so as not to leave the user asking the question "What is the valid form?"
```
    err: T('Email addresses must be entered in the format \
 <i>local-name</i>@<i>domain.com<i> with a comma \(,\) separating each \
 address.')
```